p2p/les.py

```diff
@@ -254,7 +263,7 @@ class LESProtocol(Protocol):
     _commands = [Status, Announce, BlockHeaders, BlockBodies, Receipts, Proofs, ContractCodes]
     cmd_length = 15
 
-    def send_handshake(self, head_info):
+    def send_handshake(self, head_info: Any) -> None:
```

Comment:
Should be able to use `HeadInfo` here, no?

Reply:
Ah, good point. I skipped that because I didn't notice `HeadInfo` actually exists.

Reply:
Ah, turns out it's actually `ChainInfo` there (because `genesis_hash` doesn't exist on `HeadInfo`) 

https://github.com/ethereum/py-evm/pull/947/commits/69892c3ff720ad437d76ca29d1c94df81c5380e1

Notice, I'll rebase these changes into the right place once this passed a complete review.